hermes/file_gen.go

```diff
@@ -16,21 +16,21 @@ package hermes
 import (
 	"crypto/sha1"
 	"fmt"
+	"time"
 )
 
 type HermesFile struct {
 	name     string
 	contents string
 }
 
-// method of HermesFile generates HermesFile.name of the form Hermes_id_checksum where id is an integer & id <= 50
-func (file *HermesFile) generateFileName(file_id int, file_checksum string) {
-	file.name = fmt.Sprintf("Hermes_%02d_%v", file_id, file_checksum)
+// generateFileName generates HermesFile.name of the form Hermes_id_checksum where id is an integer & id <= 50
+func generateFileName(file_id int, file_checksum string) string {
+	return fmt.Sprintf("Hermes_%02d_%v", file_id, file_checksum)
 }
 
-// method of HermesFile generates HermesFile.contents now a string without any significance in the future a pseudo random byte generator will be used
-func (file *HermesFile) generateFileContents(file_id int) {
-	file.contents = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+func generateFileContents(file_id int) string {
```

Comment:
Add a to-do comment in the link above mentioning how you will add support to a pseudo random byte generator to create the contents.